spynnaker8/models/recorder.py

```diff
@@ -49,31 +49,30 @@ def _get_io(filename):
             raise Exception("file extension %s not supported" % extension)
 
     def _extract_data(self, variables, clear, annotations):
-        """ extracts data from the vertices and puts them into a neo block
+        """ extracts block from the vertices and puts them into a neo block
```

Comment:
Should this function now be called _extract_block instead of _extract_data ?

Reply:
Its a _ method so I don't care

Reply:
changed in directneo pr